target/linux/ath79/dts/qca9533_mikrotik_routerboard-lhg-2nd.dts

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-lhg-2nd", "qca,qca9533";
+	model = "MikroTik RouterBOARD LHG 2nD";
+
+	aliases {
+		led-boot = &led_user;
+		led-failsafe = &led_user;
+		led-upgrade = &led_user;
+		led-running = &led_user;
+		serial0 = &uart;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "mikrotik:blue:power";
+			gpios = <&gpio 17 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		led_user: user {
+			label = "mikrotik:green:user";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "mikrotik:green:wlan";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		rssilow {
+			label = "mikrotik:green:rssilow";
+			gpios = <&gpio 9 GPIO_ACTIVE_LOW>;
+		};
+
+		rssimediumlow {
+			label = "mikrotik:green:rssimediumlow";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+		};
+
+		rssimediumhigh {
+			label = "mikrotik:green:rssimediumhigh";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+
+		rssihigh {
+			label = "mikrotik:green:rssihigh";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
```

Comment:
This shares the same gpio as the reset button, which usually brings chaos when rssimon twiddles that LED.

Reply:
Yes, I also found it strange, but it actually works: the LED works as intended, not causing (so far) any unwanted resets, and the button works when pressed --regardless of whether the LED is on or off--. Do you advise to remove it?

Reply:
Well if it works then great. Maybe adequate support has been added in recent ath79 code: it certainly didn’t work on ar71xx (random resets)

Reply:
Well, on a second test, it doesn't work. I either got a false positive or was completely wrong (most likely). I'm commenting it.

Reply:
That's more like what I expected :)
Make sure to comment the comment; so that people know not to blindly reenable it.

Reply:
DTS:
```dts
		/* The rssihigh LED GPIO 16 is shared with the reset button, so it remains
		unregistered here to avoid conflict.

		rssihigh {
			label = "mikrotik:green:rssihigh";
			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
		};
		*/
```
and `01_leds`:
```bash
# The rssihigh LED is disabled, as it shares GPIO 16 with the reset button
# ucidef_set_led_rssi "rssihigh" "rssihigh" "mikrotik:green:rssihigh" "wlan0" "76" "100"
```